src/App.tsx

```diff
@@ -1,156 +1,237 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable no-param-reassign */
+import classNames from 'classnames';
+import React, { useCallback, useMemo, useState } from 'react';
+
+type Todo = {
+  id: number;
+  title: string;
+  isEditing: boolean;
+};
+
+type Filter = 'All' | 'Completed' | 'Active';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completedTodos, setCompletedTodos] = useState<Todo[]>([]);
+  const [inputValue, setInputValue] = useState('');
+  const [filter, setFilter] = useState<Filter>('All');
```

Comment:
```suggestion
  const [filter, setFilter] = useState<Filter>(Filter.All);
```
always use enum value